sources/AutoFilerTab.cpp

```diff
@@ -226,12 +235,14 @@ AutoFilerTab::MessageReceived(BMessage* msg)
 				break;
 
 			BStringItem* item = (BStringItem*)fFolderList->ItemAt(selection);
-			fFilePanel->SetPanelDirectory(item->Text());
+			fEditPanel->SetPanelDirectory(item->Text());
 
 			BMessage panelMsg(MSG_FOLDER_CHOSEN);
 			panelMsg.AddInt32("index", selection);
-			fFilePanel->SetMessage(&panelMsg);
-			fFilePanel->Show();
+			fEditPanel->SetMessage(&panelMsg);
+			fEditPanel->Window()->SetTitle("AutoFiler: Edit folder");
+			fEditPanel->SetButtonLabel(B_DEFAULT_BUTTON, "Choose");
```

Comment:
The SetTitle() and SetButtonLabel() could be done once right after creating the file panels, right?